src/coreclr/jit/CMakeLists.txt

```diff
@@ -626,7 +626,14 @@ endif()
 if (CLR_CMAKE_TARGET_WIN32 AND CLR_CMAKE_PGO_INSTRUMENT)
   # Copy PGO dependency to target dir
   set(PGORT_DLL "pgort140.dll")
-  find_path(PGORT_DIR ${PGORT_DLL} REQUIRED)
+  cmake_path(SET PGORT_DIR_ARM64 "$ENV{VCToolsInstallDir}bin/arm64")
```

Comment:
I think this first assignment is dead code.

Reply:
Yeah, that was from the hint path experiment. I will remove that.